solutions/src/main/java/ru/sberbank/school/task02/exception/FxConversionServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package ru.sberbank.school.task02.exception;
+
+import lombok.NonNull;
+import ru.sberbank.school.task02.ExternalQuotesService;
+import ru.sberbank.school.task02.FxConversionService;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class FxConversionServiceImpl implements FxConversionService {
+  private ExternalQuotesService externalQuotesService;
+
+  private ClientOperation operation;
+
+  public FxConversionServiceImpl(ExternalQuotesService externalQuotesService) {
+    this.externalQuotesService = externalQuotesService;
+  }
```

Comment:
&#39;ctor def rcurly&#39; has incorrect indentation level 2, expected level should be 4.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-201743326>*